cnd/src/config/settings.rs

```diff
@@ -14,7 +15,7 @@ use std::{
 /// meaning in cnd. Contrary to that, many configuration values are optional in
 /// the config file but may be replaced by default values when the `Settings`
 /// are created from a given `Config`.
-#[derive(Clone, Debug, PartialEq)]
+#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
```

Comment:
Why do you need `Deserialize`?

Reply:
Will check, I think I added this to satisfy the compiler.

Reply:
In this case it'd be good to functionally understand whether it's actually needed.